src/garage/tf/_functions.py

```diff
@@ -0,0 +1,99 @@
+"""Utility functions for tf-based Reinforcement learning algorithms."""
+import numpy as np
+
+from garage.misc import tensor_utils as np_tensor_utils
+from garage.tf.misc import tensor_utils
+
+
+def paths_to_tensors(paths, max_path_length, baseline_predictions, discount,
+                     gae_lambda):
+    """Return processed sample data based on the collected paths.
+
+    Args:
+        paths (list[dict]): A list of collected paths.
+        max_path_length (int): Maximum length of a single rollout.
+        baseline_predictions(numpy.ndarray): : Predicted value of GAE
+            (Generalized Advantage Estimation) Baseline.
+        discount (float): Environment reward discount.
+        gae_lambda (float): Lambda used for generalized advantage
+            estimation.
+
+    Returns:
+        dict: Processed sample data, with key
+            * observations: (numpy.ndarray)
```

Comment:
does this return numpy arrays or tf.tensors? If it returns numpy arrays, can't you just use `garage.np.paths_to_tensors`?